sql/views.sql

```diff
@@ -38,7 +40,7 @@ CREATE OR REPLACE VIEW timescaledb_information.hypertables AS
                       GROUP BY hypertable_id) dn
     ON ht.id = dn.hypertable_id
     WHERE ht.compressed is false --> no internal compression tables
```

Comment:
It's not direct part of PR, but can you fix capitalisation of the keywords, since it is related and was overlooked in the correct PR review?

Reply:
This issue is prevalent throughout the SQL code. I suggest we adopt a formatter and make sure this is fixed in all places.

Reply:
This is next to your line, so it was an obvious and easy fix, taking in account that you were fixing commit message and rebase in any case. Postponing is also fine (it just looked to me weird to have two lines next to each other in different style).